### PR TITLE
VariableInput - onChange prop

### PR DESCRIPTION
**Pull request by tomershk**
### 🔦 Summary

VariableInput - onChange prop

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] API description
- 🔬 Change is tested
  - [x] Component

### CODE REVIEW

File: src/VariableInput/VariableInput.uni.driver.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default (base, body) => {
 
       // TODO: implement for puppeteer. Throw error if type is not handled
       if (base.type === 'react') {
-        ReactBase(base).enterText({ data: text });
+        Simulate.beforeInput(contentElement, { data: text });
```
Comment (nitayneeman) [inline src/VariableInput/VariableInput.uni.driver.js]: Why not using ReactBase? 

We can pass the `contentElement` if needed

Comment (tomershk) [reply src/VariableInput/VariableInput.uni.driver.js]: couldn't manage to make it work as expected
Tried: `ReactBase(contentElement).beforeInput({ data: text });`
But test are failing, as method not being called, any suggestions?

Comment (nitayneeman) [reply src/VariableInput/VariableInput.uni.driver.js]: I think you want to pass the Unidriver content element, right?

Something like:
```suggestion
        ReactBase(base.$('.public-DraftEditor-content')).beforeInput({ data: text });
```

Or even better:
```suggestion
        ReactBase(getContent(base)).beforeInput({ data: text });
```

Comment (tomershk) [reply src/VariableInput/VariableInput.uni.driver.js]: Added a TODO for this, as discussed

**Pull request merged**